README.md

```diff
@@ -1,3 +1,7 @@
+# a forked version of serverless-offline-sns to support lambda events defined using the sqs-from-sns plugin
+
+Deployments are currently manual and not run through CI
```

Comment:
Probably worth adding a CI job before merging it.

Reply:
what do you think we should do about versioning? I don't really want us to start auto incrementing this package's version

Reply:
i have created our own versioning